setup/teardown.sh

```diff
@@ -1,2 +1 @@
-conda activate base
-conda env remove --name emissioneval
+conda deactivate && conda env remove --name emission
```

Comment:
```suggestion
conda deactivate && conda env remove --name emissioneval
```

Those of us who also write server code already have an `emission` environment set up